project-base/src/Shopsys/ShopBundle/Resources/views/Front/Content/Brand/list.html.twig

```diff
@@ -13,8 +13,10 @@
                 {% for brand in brands %}
                     <li class="list-images__item">
                         <a href="{{ url('front_brand_detail', { id: brand.id }) }}" class="list-images__item__block list-images__item__block--with-label">
```

Comment:
I think here is undefined class definition named `list-images__item__block--with-label`. You removed it from `images.less`. Could you remove it here too, please? :)

Reply:
Thats true. Removed